sdcm/nemesis.py

```diff
@@ -63,6 +64,7 @@
 from sdcm.sct_events.group_common_events import (ignore_alternator_client_errors, ignore_no_space_errors,
                                                  ignore_scrub_invalid_errors)
 from sdcm.db_stats import PrometheusDBStats
+from sdcm.utils.replication_strategy_utils import KeyspaceReplicationStrategy, temporary_replication_strategy_setter
```

Comment:
well, it will be temporary only in case you revert it, but it is indeed a `setter`

Reply:
this is temporary - it automatically reverts replication strategies to original ones upon __exit__()